<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStaff</class>
 <widget class="QMainWindow" name="addStaff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加员工</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cardID">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_address">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phone">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>卡号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>性别:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>出生日期:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>身份证号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>电话:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>住址:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_sex">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>女</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_year">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1970</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1971</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1972</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1973</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1974</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1975</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1976</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1978</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1979</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1980</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1981</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1982</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1983</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1984</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1985</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1986</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1987</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1988</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1989</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1990</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1991</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1992</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1993</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1994</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1995</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1996</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1997</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1998</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1999</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_month">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>部门:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>职位:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_department">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>董事会</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>销售部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>技术部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>后勤部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_position">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>董事长</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>总经理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>经理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>副经理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>员工</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>月</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>年</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>416</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>照片:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_photo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_look">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>9</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string> *</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string> *</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string> *</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string> *</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string> *</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string> *</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string> *</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string> *</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string> *</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>369</y>
      <width>120</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);</string>
    </property>
    <widget class="QPushButton" name="pushButton_local">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>96</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>选择本地照片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cameraca">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>96</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>拍照</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_camera" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>301</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);</string>
    </property>
    <widget class="QPushButton" name="pushButton_takephoto">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>拍照</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_photoview">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>221</width>
       <height>291</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>照片</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_OK">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>310</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_retake">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>重拍</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
